### PR TITLE
Otf tablegen

### PR DESCRIPTION
**Pull request by FlanFlanagan**
Stuff for printing out the Bright-lite table each time step.

### CODE REVIEW

File: xsgen/openmc_origen.py [hunk 1]
```diff
@@ -229,7 +229,7 @@ def generate_run(self, run):
             'E_g': {'EAF': self.eafds.src_group_struct,
                     'OpenMC': self.omcds.src_group_struct},
             'phi_g': []}, 
-          "fuel": {
+            "fuel": {
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: I don't think that this indentation is correct.  It was right how it was.

File: xsgen/openmc_origen.py [hunk 2]
```diff
@@ -778,5 +778,6 @@ def _origen(origen_params):
     if burnup < 0.0:
         msg = 'Negative burnup found for {0}:\n{1}'
         msg = msg.format(mat_id, pformat(results[1]))
-        raise ValueError(msg)
+        burnup = 0.0
+        #raise ValueError(msg)
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: Remove excess comment.

**Comment by scopatz** [comment]
Thanks for putting this in @FlanFlanagan. Just a couple of minor clean up issue.  Please ping when it is ready

**Comment by FlanFlanagan** [comment]
@scopatz ping

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**